Orange/classification/majority.py

```diff
@@ -56,7 +71,11 @@ def predict(self, X):
         :return: predicted value
         :rtype: vector of majority values
         """
-        return tile(self.dist, (len(X), 1))
+        probs = tile(self.dist, (len(X), 1))
+        if self.unif_maj:
```

Comment:
The if condition fails when self.unif_maj == 0.
Should be:

    if self.unif_maj is not None: